static/js/src/advantage/subscribe/react/components/Form/Feature/Feature.tsx

```diff
@@ -25,186 +25,313 @@ const Feature = () => {
-                src="https://assets.ubuntu.com/v1/842e6353-wordpress-logo.png"
-                alt="Wordpress"
-              />
-            </div>
-          </div>
+    <>
+      <Row>
+        <Col size={6}>
+          <h2>What security coverage do you need?</h2>
         </Col>
-        <Col size={3}>
+        <Col size={6}>
           <div
             className={classNames({
-              "p-card--radio--column": true,
-              "is-selected": feature === Features.pro,
+              row: true,
+              "p-divider": true,
             })}
+            data-testid="wrapper"
           >
-            <label className="p-radio u-align-text--center">
-              <input
-                data-testid="pro"
-                className="p-radio__input"
-                autoComplete="off"
-                type="radio"
-                aria-labelledby={`pro-label`}
-                value={Features.pro}
-                checked={feature === Features.pro}
-                onChange={handleChange}
-                disabled={proDisabled}
-              />
-              <span className="p-radio__label" id={`pro-label`}>
-                <RadioInput
-                  labelClassName="inner-label"
-                  label={"Ubuntu Pro"}
-                  checked={feature === Features.pro}
-                  value={Features.pro}
-                  onChange={handleChange}
-                  disabled={proDisabled}
-                />
-                <div className="included">
-                  <i className="p-icon--success"></i>Included
-                </div>
-                <div className="included">
-                  <i className="p-icon--success"></i> Included
-                </div>
-              </span>
-            </label>
+            <p>
+              Not sure? Run <code> pro security-status</code> to find out which
+              packages and repositories you are currently using.
+            </p>
+            <RadioInput
+              label="Pro - all repositories"
+              checked={feature === Features.pro}
+              value={Features.pro}
+              onChange={handleChange}
+              disabled={proDisabled}
+              data-testid="pro"
+            />
+            <hr />
+            <RadioInput
+              label="Infra only - limited subset"
+              checked={feature === Features.infra}
+              value={Features.infra}
+              onChange={handleChange}
+              disabled={infraOnlyDisabled}
+              data-testid="infra-only"
+            />
           </div>
         </Col>
+        <Col size={11} emptyMedium={2} emptyLarge={2}>
+          <hr className="is-fixed-width" />
+        </Col>
+      </Row>
+      <Row>
+        <Col size={2} emptyMedium={2} emptyLarge={2}>
+          <h1>{feature === Features.pro ? "25000" : "2300"}</h1>
+        </Col>
         <Col size={3}>
+          <p>
+            Covers {feature === Features.pro ? "25000" : "2300"} packages in
```

Comment:
This whole thing can become just `packages covered` since we have the descriptions on the graph labels.